src/skywallet/transaction.go

```diff
@@ -15,9 +15,23 @@ import (
 
 // TransactionSigner represents a data (inputs, outputs) needed to sign transaction
 type TransactionSigner interface {
+	SetDevice(*Device)
```

Comment:
dont think we need this here
the api that `TransactionSigner` "exposes" should be related to signing methods only

Reply:
I have made it to be able to call [GeneralTransactionSign](https://github.com/SkycoinProject/hardware-wallet-daemon/blob/23393e20a7582621f7ad466162ea162a49ae2d5c/src/api/bitcoin_transaction_sign.go#L94) using interface Gatewayer in hardware-wallet-daemon. So if a Device was not known on time of creating Signer, it can be setted later